tests/optim/test_oss.py

```diff
@@ -811,9 +817,11 @@ def check_optimizer_equivalence(optimizer: Type[torch.optim.Optimizer], change_t
 
         # Define a model to be trained by OSS
         oss_module = torch.nn.Sequential(trunk, head)
+
+        # Make sure that the param groups are interleaved, to catch an ordering bug in the state dict
         oss_trainable_params = [
-            {"params": trunk.parameters(), "lr": 1e-5},
```

Comment:
this unit test let a "subtle" bug pass: it was testing param groups which are sequential with respect to the model parameters, turns out that there was a bug if this was not the case.